pkg/handler/project/project.go

```diff
@@ -1564,12 +1564,12 @@ func (h *handler) CreateWorkUnit(c *gin.Context) {
 
 	// create work unit member
 	for _, employee := range employees {
-		_, err = h.store.ProjectMember.One(tx.DB(), input.ProjectID, employee.ID.String(), false)
-		if errors.Is(err, gorm.ErrRecordNotFound) {
-			l.Error(errs.ErrMemberIsNotActiveInProject, "member is not active in project")
-			c.JSON(http.StatusInternalServerError, view.CreateResponse[any](nil, nil, done(errs.ErrMemberIsNotActiveInProject), input, ""))
-			return
-		}
+		// _, err = h.store.ProjectMember.One(tx.DB(), input.ProjectID, employee.ID.String(), false)
+		// if errors.Is(err, gorm.ErrRecordNotFound) {
+		// 	l.Error(errs.ErrMemberIsNotActiveInProject, "member is not active in project")
+		// 	c.JSON(http.StatusInternalServerError, view.CreateResponse[any](nil, nil, done(errs.ErrMemberIsNotActiveInProject), input, ""))
+		// 	return
+		// }
```

Comment:
keep it, we add a member to the work unit when a member exists in the project

Reply:
but the function one only get the active member, so we can't add the inactive member to the workunit